### DISCUSSION TITLE
HELP! Unable to reach nested modules

### DISCUSSION THREAD

**Discussion by h2oearth** [terraform-linters/tflint#2123] [Q&A]
This is my `.tlint.hcl`

```
plugin "aws" {
    enabled = true
    version = "0.32.0"
    source  = "github.com/terraform-linters/tflint-ruleset-aws"
}

plugin "terraform" {
    enabled = true
    version = "0.9.1"
    preset = "recommended"
    source  = "github.com/terraform-linters/tflint-ruleset-terraform"
}

rule "terraform_comment_syntax" {
    enabled = true
}

rule "terraform_unused_required_providers" {
    enabled = true
}

rule "terraform_standard_module_structure" {
    enabled = true
}

rule "terraform_documented_outputs" {
    enabled = true
}

rule "terraform_documented_variables" {
    enabled = true
}

rule "terraform_standard_module_structure" {
    enabled = true
}

```

This is my directory structure
```
├── modules
│   ├── cloudfront
│   │   ├── main.tf
│   │   └── variables.tf
│   ├── s3
│   │   ├── main.tf
│   │   ├── output.tf
│   │   └── variables.tf

```

I am using the latest tflint version (0.53) with the following parameters `tflint --call-module-type=local --color --recursive -f compact` I tired `--call-module-type=none` and `--call-module-type=all` with the same result :-/. BTW I am using the latest AWS provider (5.66) and Terraform (1.95)

tflint can only reach root, but not the modules. I introduced some terraform linter violations on some modules to test it. Am I missing something is this a 🐛 ?

Cheers

**Comment by wata727** [comment]
Normally, `--recursive` also inspects nested modules. This behavior is tested in CI so it's probably not a bug.
If you think it's a bug, please provide the minimum code to reproduce the problem.
